Code/Framework/AzFramework/AzFramework/Asset/AssetSystemComponent.cpp

```diff
@@ -167,6 +167,58 @@ namespace AzFramework
                 }
             });
 
+            m_bulkMessageHandle = m_socketConn->AddMessageHandler(
+                BulkAssetNotificationMessage::MessageType,
+                [context](unsigned int /*typeId*/, unsigned int /*serial*/, const void* data, unsigned int dataLength)
+                {
+                    if (dataLength)
+                    {
+                        BulkAssetNotificationMessage bulkMessage;
+
+                        // note that we forbid asset loading and we set STRICT mode.  These messages are all the kind of message that is
```

Comment:
Not quite sure what "Note: We forbid asset loading and we set strict mode." has to do with bulk asset notification processing.

Reply:
Hm, I copied it from the regular message.  The bulk one is just a wrapper around the regular message.  I think perhaps without explicitly disabling the asset loading, the objectstream might see the AssetIds and try to load them as assets, so that might be what that is about.  And strict mode I think is related to error tolerance when parsing, which is set since this is not a message coming from a file a user can mess up.  I think the comment does still apply here.